package.json

```diff
@@ -15,7 +15,7 @@
     "bcrypt": "^5.1.1",
     "dayjs": "^1.11.13",
     "dotenv-safe": "^9.1.0",
-    "next": "15.0.0-canary.171",
+    "next": "15.0.3",
```

Comment:
@Eprince-hub I guess this change (and the lockfile change) is not necessary in this PR, right?

Reply:
Reverted the version to what it was and no diff lock file as well
9e8064f816cd05fcbdfdb21bf841dffb5196691d